src/test/kotlin/com/tylerthrailkill/helpers/prettyprint/CollectionsTest.kt

```diff
@@ -7,6 +7,25 @@ import java.io.ByteArrayOutputStream
 object CollectionsTest : Spek({
     setup()
 
+    describe("pretty printing array should") {
+        it("render an array") {
+            prettyPrint(Array(1){ 0 }) mapsTo """
+                [
+                  0
+                ]
+                """
+        }
+        it("render an array of array") {
+            prettyPrint(Array(1){Array(1) { 0 } }) mapsTo """
+                [
+                  [
+                    0
+                  ]
+                ]
+                """
+        }
+    }
+
```

Comment:
can you add another test that test arrays with multiple values? and one that tests with null values?